code-challenges.test.js

```diff
@@ -63,3 +129,33 @@ var vowelTester3 = "challenge"
 
 
 // b) Create the function that makes the test pass.
+
+// Well, I'm jumping into this knowing what I have to do, but not entirely sure how I am going to get there. I tried using filter thinking it is going to return an array smaller than the initial array. But in order for it to work I have to split the string into an array. I figure I can write out each vowel and using the logical operator "or".
+
+// const firstVowel = (string) => {
+//   var string1 = string.split("")
+//    var string2 = string1.filter((value, index) => {
+//     if(value === "a" || value === "e" || value === "i" || value === "o" || value ==="u"){ return index
+//     }
+//   })
+//   // return string2
+//   console.log(string2)
+// }
+// Well that was a fail. I got a big fat red with it claiming syntax error on filter. I did do a log on string2 and it just returned the vowels so it basically did not work. Maybe I am over thinking this (which is my typical m.o.) so I am going to switch back to my best friend a basic for loop.
+// I am going to see if I can loop through and use charAt for each index and use the logical operator to comapare once again to the vowels.
+// Finally I will return the index and see what happens.
+
+
+const firstVowel = (string) => {
+  for (let i = 0; i < string.length; i ++){
+    if(string.charAt(i) === "a" || string.charAt(i) === "e" || string.charAt(i) === "i" || string.charAt(i) === "o" || string.charAt(i) === "u"){
+      return i
+    }
+  }
+}
+
+// Yesssss!!! I know testing works, but it's still a little unbelievable so I am going to console.log just in case...
+
+console.log(firstVowel(vowelTester3))
+
+// ...and it totally works. I guess I can learn a few things from my over thinking... don't over think it. 
```

Comment:
for loop = screwdriver  filter = impact driver. Sometimes the for loop is all you need